src/app/clients/ClientsPage.cy.tsx

```diff
@@ -78,36 +85,27 @@ const sampleRawExpandedClientDetails: RawClientDetails = {
 describe("Clients Page", () => {
     describe("Backend Processing for Table Data", () => {
         it("Fields are set correctly", () => {
-            cy.intercept("POST", "**/check-congestion-charge", (req) => {
-                const response = req.body.postcodes.map((postcode: string) => ({
-                    postcode,
-                    congestionCharge: true,
-                }));
-
-                req.reply(JSON.stringify(response));
-            });
-
-            cy.wrap(null).then(() =>
-                processingDataToClientsTableData(sampleProcessingData).then((clientTableData) => {
-                    expect(clientTableData).to.deep.equal([
-                        {
-                            primaryKey: "PRIMARY_KEY_1",
-                            parcelId: "PRIMARY_KEY",
-                            flaggedForAttention: false,
-                            requiresFollowUpPhoneCall: true,
-                            fullName: "CLIENT_NAME",
-                            familyCategory: "Family of 3",
-                            addressPostcode: "SW1A 2AA",
-                            collectionCentre: "COLLECTION_CENTRE",
-                            collectionDatetime: "2023-08-04T13:30:00+00:00",
-                            congestionChargeApplies: true,
-                            packingDate: "04/08/2023",
-                            packingTimeLabel: "PM",
-                            lastStatus: "LAST_EVENT @ 04/08/2023",
-                        },
-                    ]);
-                })
+            const clientTableData = processingDataToClientsTableData(
+                sampleProcessingData,
+                sampleCongestionChargeData
             );
+            expect(clientTableData).to.deep.equal([
```

Comment:
How come we can't just have equal here?

Reply:
I think 'shallow' equals will check if they are the same object in memory, which they aren't here (because they are created at two different points) - whereas 'deep' equals checks each property recursively, meaning two objects are equal if they 'look' the same - which is what we want here

Reply:
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/104017547/3d1fd537-6eb6-4fdb-83dc-431310e2ad84)
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/104017547/a38883b0-6696-4d2e-acc8-c72682edd895)